react-native-git-upgrade/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-native-git-upgrade",
+  "version": "0.1.0",
+  "license": "BSD-3-Clause",
+  "description": "The React Native upgrader tool",
+  "main": "cli.js",
+  "bin": {
+    "react-native-git-upgrade": "index.js"
+  },
+  "engines": {
+    "node": ">=4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/react-native.git"
+  },
+  "scripts": {
+    "test": "mocha"
```

Comment:
No test in the meantime.

The core of the tool seems hard to test, it relies on external commands, filesystem operations,... 

I planned to dig into that... later 😉